App/iOS/Delegates/AppDelegate.swift

```diff
@@ -33,7 +33,7 @@ extension AppDelegate {
 
 class AppDelegate: UIResponder, UIApplicationDelegate {
   var window: UIWindow?
-  var braveCore: BraveCoreMain = {
+  lazy var braveCore: BraveCoreMain = {
```

Comment:
why do we decide to lazily initialize braveCore ?

Reply:
@soner-yuksel It will still be initialized before `init` is called below without lazy.  Other option is to move the initialization to `init` method after app constants set.